karabo_bridge/simulation.py

```diff
@@ -327,7 +321,8 @@ def start_gen(port, ser='msgpack', version='2.2', detector='AGIPD',
                 if n % TIMING_INTERVAL == 0:
                     t_now = time()
                     print("Sent {} trains in {:.2f} seconds ({:.2f} Hz)".format(
-                        TIMING_INTERVAL, t_now - t_prev, TIMING_INTERVAL / (t_now - t_prev)
+                        TIMING_INTERVAL, t_now -
+                        t_prev, TIMING_INTERVAL / (t_now - t_prev)
```

Comment:
Code style nitpick, can we split this line just after one of the commas, rather than in the middle of a subtraction?